src/AutoFixture.AutoEF.Tests/Interception/PropertyGetterInterceptionPolicyTest.cs

```diff
@@ -44,6 +44,27 @@ public void ReturnsFalseOnSetter(SUT sut, IInvocation invocation)
             var result = sut.ShouldIntercept(invocation);
 
             result.Should().BeFalse();
+        }
+
+
+        [Theory, AutoNSub]
+        public void ReturnsTrueOnGetterWhenTableNameId(SUT sut, IInvocation invocation)
+        {
+            invocation.Method.Returns(typeof(Far).GetProperty("Boo").GetGetMethod());
+
+            var result = sut.ShouldIntercept(invocation);
+
+            result.Should().BeTrue();
+        }
+
+        [Theory, AutoNSub]
+        public void ReturnsFalseOnSetterWhenTableNameId(SUT sut, IInvocation invocation)
+        {
+            invocation.Method.Returns(typeof(Far).GetProperty("Boo").GetSetMethod());
+
+            var result = sut.ShouldIntercept(invocation);
+
+            result.Should().BeFalse();

```

Comment:
I'm not sure these tests need to exist - it looks like the only changed functionality is in `IdPropertySetterInterceptor` and `ParentPropertySetterInterceptor`, in which case the `PropertyGetterInterceptionPolic` will not have been touched